PlaybackLab/TVOSSwiftSampleApp/TVOSSwiftSampleApp.xcodeproj/project.pbxproj

```diff
@@ -463,12 +465,8 @@
 				PRODUCT_BUNDLE_IDENTIFIER = com.ooyala.TVOSSwiftSampleApp;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SWIFT_OBJC_BRIDGING_HEADER = "$(PROJECT_DIR)/TVOSSwiftSampleApp/Supporting Files/TVOSSwiftSampleApp-Bridging-Header.h";
-				SWIFT_VERSION = 4.2;
+				SWIFT_VERSION = 5.0;
```

Comment:
@maxkupetskii  FYI: be aware that I won't be able to launch this TVOSSwiftSampleApp until I update my Xcode. Now my macOS don't allow me to update Xcode. The same for all SApps, that migrated to Swift-5.0

Reply:
It's not a macOS that prevents you from upgrade, it's IT service :)